BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.m

```diff
@@ -107,6 +107,8 @@
 BOXSharedLinkAccessLevel *const BOXSharedLinkAccessLevelOpen = @"open";
 BOXSharedLinkAccessLevel *const BOXSharedLinkAccessLevelCompany = @"company";
 BOXSharedLinkAccessLevel *const BOXSharedLinkAccessLevelCollaborators = @"collaborators";
+BOXSharedLinkAccessLevel *const BOXSharedLinkExpireLinkFileAccess = @"expireLinkFiles";
```

Comment:
This constant name doesn't actually match the definition above, and is not used anywhere. Can we just remove these?

Reply:
removed